Sources/PortalSwift/Core/mpc/PortalMpc.swift

```diff
@@ -340,7 +340,69 @@ public class PortalMpc {
       }
     }
   }
+    
+    /// Uses the org and client backup shares to return the private key
+    ///  - Parameters:
+    ///    - cipherText: the cipherText of the client's backup share
+    ///    - method: The specific backup storage option.
+    ///    - orgShare: the stringified version of the organization's backup share
+    public func ejectPrivateKey(
```

Comment:
I think this function should be called something more descriptive, since this is the function customers will actually be invoking. This should mirror the MPC functions with something like `ejectPrivateKeyAndDiscontinueMPC`.

Reply:
Handled over standup call